app/controllers/gigs.js

```diff
@@ -176,12 +176,45 @@ module.exports = function(app) {
           "Authorization": GROOT_ACCESS_TOKEN
         },
         json: true,
-        body: {}
+        body: {
+          active: false
+        }
       }, function(err, response, body) {
         return res.send(body);
       });
     });
   });
+  app.post('/intranet/gigs/edit/:gig_id', function(req, res) {
+    request({
+      url: `${SERVICES_URL}/gigs/${req.params.gig_id}`,
+      method: "GET",
+      headers: {
+        "Authorization": GROOT_ACCESS_TOKEN,
+      },
+      json: true
+    }, function(err, response, body) {
+      if(body.issuer != req.session.student.netid) {
```

Comment:
May want to implement that default error handling that we tend to do for requests i.e 

```
if (err || !response || body.statusCode != 200) {

}
```

Honestly, it may be worth considering to make these request blocks a helper function (not in this PR, just in general) that has this error check already so we can avoid a lot of code duplication and make our actual controller logic a bit shorter.